dependency/dependency_sqlite.go

```diff
@@ -0,0 +1,80 @@
+package dependency
+
+import (
+	"database/sql"
+
+	"github.com/vacp2p/mvds/state"
+)
+
+// Verify that MessageDependency interface is implemented.
+var _ MessageDependency = (*sqliteDependency)(nil)
+
+type sqliteDependency struct {
+	db *sql.DB
+}
+
+func NewPersistentDependency(db *sql.DB) *sqliteDependency {
+	return &sqliteDependency{db: db}
+}
+
+func (sd *sqliteDependency) Add(msg, dependency state.MessageID) error {
+	_, err := sd.db.Exec(`INSERT INTO mvds_dependencies (msg, dependency) VALUES (?, ?)`, msg[:], dependency[:])
```

Comment:
```suggestion
	_, err := sd.db.Exec(`INSERT INTO mvds_dependencies (msg_id, dependency) VALUES (?, ?)`, msg_id[:], dependency[:])
```

Generally, I see `msg` in many places while we actually deal with `msgID`. For me it's confusing because I need to check whether it's actual message or only an ID.